libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -1166,6 +1186,14 @@ bool AudioIO::AllocateBuffers(
             mPlaybackQueueMinimum =
                std::min( mPlaybackQueueMinimum, playbackBufferSize );
 
+            // Limit the mPlaybackQueueMinimum to the hardware latency
+            mPlaybackQueueMinimum =
+               std::max(mPlaybackQueueMinimum, mHardwarePlaybackLatencyFrames);
+
+            // Make mPlaybackQueueMinimum a multiple of mPlaybackSamplesToCopy
+            mPlaybackQueueMinimum = mPlaybackSamplesToCopy *
+               ((mPlaybackQueueMinimum + mPlaybackSamplesToCopy - 1) / mPlaybackSamplesToCopy);
```

Comment:
This last assignment may make `mPlaybackQueueMinimum` still larger than `mHardwarePlaybackLatencyFrames`

Is that what you want?  Or should it be

`mPlaybackQueueMinimum -= mPlaybackQueueMinimum % mPlaybackSamplesToCopy`

Reply:
It should be `not less`, so larger definitely works, otherwise bug won't be fixed.

Reply:
Oh I was being silly

Reply:
Still, in two places, one thing is rounded to a multiple of another thing.  I don't yet understand the need.